sdk/servicebus/azure-servicebus/azure/servicebus/_common/utils.py

```diff
@@ -24,7 +24,7 @@
     cast
 )
 from contextlib import contextmanager
-from msrest.serialization import TZ_UTC
+from azure.core.serialization import TZ_UTC
```

Comment:
Python 3:
``` from datetime import timezone
TZ_UTC = timezone.utc
```
Or even you remove `TZ_UTC` and replace them all with `timezone.utc`